{
  try {
    final TestOne testOne=createTestNode(TestOne.class);
    final TestSix testSix=createTestNode(TestSix.class);
    SixOneOneToOne rel=null;
    assertNotNull(testOne);
    assertNotNull(testSix);
    try (final Tx tx=app.tx()){
      rel=app.create(testSix,testOne,SixOneOneToOne.class);
      tx.success();
    }
     assertNotNull(rel);
    try {
      rel.getRelationship().delete();
      fail("Should have raised an org.neo4j.graphdb.NotInTransactionException");
    }
 catch (    NotInTransactionException e) {
    }
    assertNotNull(rel);
    try (final Tx tx=app.tx()){
      app.delete(rel);
      tx.success();
    }
     try (final Tx tx=app.tx()){
      String uuid=rel.getUuid();
      fail("Deleted entity should have thrown an exception on access.");
    }
 catch (    NotFoundException iex) {
    }
  }
 catch (  FrameworkException ex) {
    logger.log(Level.WARNING,"",ex);
    logger.log(Level.SEVERE,ex.toString());
    fail("Unexpected exception");
  }
}
